### PR TITLE
User/le bat/add signup login

### PR DESCRIPTION
**Pull request by coffeefee-coder**
Feel free to give suggestions on the formatting and especially the angular(since it's my first time really working with it). I've added functions that will handle the information and where to send it but they don't do anything at the moment. Wasn't entirely sure if we had that fully set up yet.

### CODE REVIEW

**Comment by ghost** [comment]
Looks fine to me. When it comes to submitting requests to the API, check this page out and scroll down to "Shortcut methods": https://docs.angularjs.org/api/ng/service/$http

File: assets/js/app.js [hunk 1]
```diff
@@ -58,6 +66,31 @@ app.controller('AboutCtrl', ['$scope', '$rootScope', function($scope, $rootScope
...
+
+app.controller('SignupCtrl', ['$scope', '$rootScope', function($scope, $rootScope) {
+  $rootScope.root = {
+    route: 'Sign Up',  // this corresponds to the menu item that should be active
+    title: 'Sign Up | GitSubmit'
+  }
+
+    app.controller('FormSignupCtrl', ['$scope', function($scope) {
+      $scope.submit = function() {
+        //TODO:Add code to handle the fields and send to approriate destination
```
Comment (sonph) [inline assets/js/app.js]: I'd suggest adding a test method that shows the values entered e.g. `alert($scope.email + ', ' + $scope.username + ...);`

File: assets/js/app.js [hunk 2]
```diff
@@ -58,6 +66,31 @@ app.controller('AboutCtrl', ['$scope', '$rootScope', function($scope, $rootScope
...
 }])
 
+app.controller('LoginCtrl', ['$scope', '$rootScope', function($scope, $rootScope) {
+  $rootScope.root = {
+    route: 'Login',  // this corresponds to the menu item that should be active
+    title: 'Login | GitSubmit'
+  }
+  app.controller('FormLoginCtrl', ['$scope', function($scope) {
+      $scope.submit = function() {
+        //TODO:Add code to handle the fields and send to approriate destination
```
Comment (sonph) [inline assets/js/app.js:77]: Same thing here, probably we should add a method showing the values (until the backend works then we'll hook this up with it).

File: login.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
+    <div class="spacer100"></div>
+    <h1>Login!</h1>
+    <form ng-submit="submit()" ng-controller="FormLoginCtrl">
+        Username:<br>
+        <input type="text" ng-model="username"><br>
+        Password:<br>
+        <input type="password" ng-model="email"><br>
```
Comment (sonph) [inline login.html]: `ng-model` should be `password`

**Comment by sonph** [comment]
Just a couple of things I noted. Otherwise great work :+1: :+1:

File: signup.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    <h1>Sign Up!</h1>
+    <div>
+        <form ng-submit="submit()" ng-controller="FormSignupCtrl">
+            Email:<br>
+            <input type="email" ng-model="email"><br>
+            Username:<br>
+            <input type="text" ng-model="username"><br>
+            Password:<br>
+            <input type="password" ng-model="password"><br>
+            <button ng-click="submit()">Submit</button>
```
Comment (sonph) [inline signup.html]: For these submit buttons you can style it with materialize (see [this link on button, especially the submit button with icon](http://materializecss.com/buttons.html))

**Comment by coffeefee-coder** [comment]
Thanks for the feedback! I'll look into those things.

**Comment by Spencatro** [comment]
conditional +1 from me, after the comments above are addressed

**Comment by ghost** [comment]
+1

**Comment by coffeefee-coder** [comment]
I made the changes that were advised and will go ahead and merge this now

**Pull request merged**